test/cmd/authentication.sh

```diff
@@ -38,6 +38,9 @@ os::cmd::expect_success 'oc get secrets --token="${allescalatingpowerstoken}" -n
 os::cmd::expect_failure_and_text 'oc get secrets --token="${allescalatingpowerstoken}" -n default' 'cannot list secrets in project'
 os::cmd::expect_success_and_text 'oc get projects --token="${allescalatingpowerstoken}"' 'cmd-authentication'
 
+accesstoken=$(oc process -f ${OS_ROOT}/test/fixtures/authentication/scoped-token-template.yaml TOKEN_PREFIX=access SCOPE=user:check-access USER_NAME="${username}" USER_UID="${useruid}" | oc create -f - -o name | awk -F/ '{print $2}')
```

Comment:
So if you tried to get this user's info using this scope, that should fail, right? Or is check-access a superset of user:info?